elyra/pipeline/processor_airflow.py

```diff
@@ -169,6 +169,14 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                                                    cos_username=cos_username,
                                                    cos_password=cos_password)
 
+                # Generate unique ELYRA_RUN_NAME value and expose it as an
+                # environment variable in the container
+                # https://airflow.apache.org/docs/apache-airflow/1.10.12/_api/airflow/contrib/operators/kubernetes_pod_operator/index.html
+                # https://airflow.apache.org/docs/apache-airflow/1.10.12/macros-ref#default-variables
```

Comment:
I see `run_id` referenced in the second link, but I'm not able to determine what the first link is about.  Should we add some kind of description for how these links are helpful?

Reply:
Done